templates/boltzmann.js

```diff
@@ -575,7 +579,105 @@ function applyHeaders (headers = {}) {
   }
 }
 
-const applyXFO = (mode) => applyXFO.bind({ 'x-frame-options': mode })
+const applyXFO = (mode) => applyHeaders({ 'x-frame-options': mode })
+
+// {% if csrf %}
+// csrf protection middleware
+function signCookie(value, secret) {
+  return `${value}.${crypto.createHmac('sha256', secret).update(value).digest('base64')}`
+}
```

Comment:
🔴 this...

Reply:
I had been holding off on cookie signing because I want to take a closer look at the steps Iron takes to sign. (I looked at this with fresh eyes a little bit ago for ormnomnom – a nonce is involved, notably.) However, once we get there, it'd probably be good to apply to both places, so it's totally fine to leave this function in.